tests/test_ingesters.py

```diff
@@ -483,6 +490,8 @@ def setUp(self):
         self.mock_requests_get = self.patcher_requests_get.start()
         self.mock_requests_get.side_effect = self.requests_get_side_effect
         self.opened_files = []
+        Parameter.objects.count = mock.Mock()
+        Parameter.objects.count.return_value = 2
```

Comment:
Please use patch for this, in order to limit unintended side effects

Reply:
Done in the next commit

Reply:
Done in the next commit